app/assets/javascripts/application.js

```diff
@@ -13,5 +13,6 @@
 //= require jquery
 //= require jquery_ujs
 //= require turbolinks
+//= require expose
```

Comment:
This is intentionally above `require_tree` so it is available before any other custom javascripts